apps/admin-gui/src/app/admin/pages/admin-page/admin-searcher/admin-searcher.component.ts

```diff
@@ -0,0 +1,164 @@
+import { Component, OnInit } from '@angular/core';
+import {
+  AttributeDefinition,
+  AttributesManagerService,
+  EnrichedFacility,
+  RichMember,
+  RichUser,
+  SearcherService,
+  Vo,
+  VosManagerService,
+} from '@perun-web-apps/perun/openapi';
+import { compareFnDisplayName, compareFnName } from '@perun-web-apps/perun/utils';
+import {
+  TABLE_SEARCHER_FACILITIES,
+  TABLE_SEARCHER_MEMBERS,
+  TABLE_SEARCHER_RESOURCES,
+  TABLE_SEARCHER_USERS,
+} from '@perun-web-apps/config/table-config';
+import { ResourceWithStatus } from '@perun-web-apps/perun/models';
+
+@Component({
+  selector: 'app-admin-searcher',
+  templateUrl: './admin-searcher.component.html',
+  styleUrls: ['./admin-searcher.component.scss'],
+})
+export class AdminSearcherComponent implements OnInit {
+  constructor(
+    private attributesManager: AttributesManagerService,
+    private searcher: SearcherService,
+    private voService: VosManagerService
+  ) {}
+
+  loading: boolean;
+  loadingEntityData = false;
+  allAttrDefinitions: AttributeDefinition[] = [];
+  availableAttrDefs: AttributeDefinition[] = [];
+  searchInput: { [p: string]: string };
+  tableId = TABLE_SEARCHER_USERS;
+  entities: RichUser[] | RichMember[] | EnrichedFacility[] | ResourceWithStatus[] = [];
+  vos: Vo[] = [];
+  selectedVo: Vo;
+
+  ngOnInit(): void {
+    this.loading = true;
+    this.attributesManager.getAllAttributeDefinitions().subscribe((attrDefs) => {
+      this.allAttrDefinitions = attrDefs;
+      this.filterAttrDefinitions('user');
+      this.loading = false;
+    });
+  }
+
+  filterAttrDefinitions(entity: string): void {
+    this.availableAttrDefs = this.allAttrDefinitions
+      .filter((attrDef) => attrDef.entity === entity)
+      .sort(compareFnDisplayName);
```

Comment:
Maybe this could be part of the attribute search select component

Reply:
This functionality has been moved to the attribute-search-select component.